test/cpp/microbenchmarks/BUILD

```diff
@@ -48,75 +48,98 @@ grpc_cc_library(
     ],
 )
 
-grpc_cc_binary(
-    name = "bm_closure",
+# Need a secure version of helpers to benchmark opencensus
+grpc_cc_library(
+    name = "helpers_secure",
     testonly = 1,
+    srcs = ["helpers.cc"],
+    hdrs = [
+        "fullstack_context_mutators.h",
+        "fullstack_fixtures.h",
+        "helpers.h",
+    ],
+    external_deps = [
+        "benchmark",
+    ],
+    tags = ["no_windows"],
+    deps = [
+        "//:grpc++",
+        "//src/proto/grpc/testing:echo_proto",
+        "//test/core/util:grpc_test_util",
+        "//test/cpp/util:test_config",
+    ],
+)
+
+grpc_cc_test(
+    name = "bm_closure",
     srcs = ["bm_closure.cc"],
     tags = ["no_windows"],
     deps = [":helpers"],
 )
 
-grpc_cc_binary(
+grpc_cc_test(
     name = "bm_alarm",
-    testonly = 1,
     srcs = ["bm_alarm.cc"],
     tags = ["no_windows"],
     deps = [":helpers"],
 )
 
-grpc_cc_binary(
+grpc_cc_test(
     name = "bm_arena",
-    testonly = 1,
+    size = "enormous",
```

Comment:
why enormous?

Reply:
I can take the enormous tests back down to large or less. I think that was when I was just experimenting. I can also disable all tests on mac.